migrations/versions/12780a039923_populate_database.py

```diff
@@ -0,0 +1,82 @@
+"""populate database
+
+Revision ID: 12780a039923
+Revises: 67ac3b7d5c2f
+Create Date: 2021-09-14 14:44:50.802230
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '12780a039923'
+down_revision = '67ac3b7d5c2f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
We need to migrate existing data here, including finding existing `User`s - who may no longer be part of RC, and thus not be in the data returned by the RC API - and populating the new `profile` table with that data.